velox/expression/tests/ExprTest.cpp

```diff
@@ -575,50 +575,9 @@ class ExprTest : public testing::Test, public VectorTestBase {
         std::make_shared<ConstantVector<T>>(execCtx_->pool(), 1, index, base));
   }
 
-  VectorPtr makeConstantVector(variant value, vector_size_t size) {
-    return BaseVector::createConstant(value, size, execCtx_->pool());
-  }
-
-  template <typename T>
-  FlatVectorPtr<T> makeFlatVector(const std::vector<T>& data) {
-    return vectorMaker_->flatVector<T>(data);
-  }
-
-  template <typename T>
-  FlatVectorPtr<T> makeFlatVector(
-      vector_size_t size,
-      std::function<T(vector_size_t /*row*/)> valueAt,
-      std::function<bool(vector_size_t /*row*/)> isNullAt = nullptr) {
-    return vectorMaker_->flatVector<T>(size, valueAt, isNullAt);
-  }
-
-  FlatVectorPtr<StringView> makeFlatVector(
-      const std::vector<std::string>& data) {
-    return vectorMaker_->flatVector(data);
-  }
-
   template <typename T>
   FlatVectorPtr<T> makeAllNullFlatVector(vector_size_t size) {
```

Comment:
Could this function be removed too?

Reply:
makeAllNullFlatVector is not available in VectorTestBase, but I can move it.

Reply:
I mean I saw makeAllNullFlatVector simply calls `vectorMaker_.allNullFlatVector<T>(size)`. So I thought we can just use vectorMaker_.allNullFlatVector() instead?